docs/docs/getting-started/first-steps.md

```diff
@@ -40,45 +40,61 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 2. Fill in your cloud provider specific information.
 
     <tabs groupId="csp">
-    <tabItem value="azure" label="Azure (CLI)">
 
-    You need several resources for the cluster. You can use the following `az` script to create them:
+    <tabItem value="azure-command" label="Azure">
+
+    Create the IAM configuration on Azure with the following command (You must be authenticated with the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) in the shell session):
 
     ```bash
-    RESOURCE_GROUP=constellation # enter name of new resource group for your cluster here
-    LOCATION=westus # enter location of resources here
-    SUBSCRIPTION_ID=$(az account show --query id --out tsv)
-    SERVICE_PRINCIPAL_NAME=constell
-    az group create --name "${RESOURCE_GROUP}" --location "${LOCATION}"
-    az group create --name "${RESOURCE_GROUP}-identity" --location "${LOCATION}"
-    az ad sp create-for-rbac -n "${SERVICE_PRINCIPAL_NAME}" --role Owner --scopes "/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}" | tee azureServiceAccountKey.json
-    az identity create -g "${RESOURCE_GROUP}-identity" -n "${SERVICE_PRINCIPAL_NAME}"
-    identityID=$(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query principalId --out tsv)
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Virtual Machine Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Application Insights Component Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    echo "subscription: ${SUBSCRIPTION_ID}
-    tenant: $(az account show --query tenantId -o tsv)
-    location: ${LOCATION}
-    resourceGroup: ${RESOURCE_GROUP}
-    userAssignedIdentity: $(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query id --out tsv)
-    appClientID: $(jq -r '.appId' azureServiceAccountKey.json)
-    clientSecretValue: $(jq -r '.password' azureServiceAccountKey.json)"
+    constellation iam create azure --region=westus --resourceGroup=constellTest --servicePrincipal=spTest
```

Comment:
Even though the reference contains detailed information and is linked, I think we could state briefly what is expected of the user.

When reading the command for the first time, I asked myself the following question: Is `resourceGroup` a name for something that is to be generated, or a reference to something that already exists. The same for `servicePrincipal`.

Reply:
I wonder whether we should have a workflow page for IAM creation and link to this instead. But it might also be too redundant and superfluous. Opinions?

Reply:
I would prefer to only briefly state the command here for users who want to set up really quickly here and moving detailed descriptions to a workflow page. There, we could also explain how the IAM resources can be re-used or deleted. Let me know your opinions.

Reply:
> moving detailed descriptions to a workflow page. There, we could also explain how the IAM resources can be re-used or deleted

A workflow page seems a good solution if we actually have additional infos there like you propose 👍

Reply:
Should we also link to the workflow pages in the getting-started guide? Currently, i think we only do this for the verification section (which is removed in v2.3 anyways?) and cluster recovery. However, i think for a user it might be nice to get a link to detailed information about the step he is currently performing, such as IAM or cluster creation. What do you think?

Reply:
If we have a dedicated workflow page for IAM it is probably better to link that than the CLI reference.

I think links to workflows are helpful if they do not interrupt the flow of the guide. I checked our quick start for this:

I like: "This will allow you to [recover your cluster](https://constellation-docs.edgeless.systems/constellation/workflows/recovery) in case of a disaster."

Less so: "For details, see the [verification section](https://constellation-docs.edgeless.systems/constellation/workflows/verify-cluster)."

Reply:
completely agree with Fabian